diff -rupN boost_1_49_0/boost/thread/xtime.hpp boost_1_49_0_new/boost/thread/xtime.hpp
--- boost_1_49_0/boost/thread/xtime.hpp	2008-06-18 15:01:08.000000000 +0200
+++ boost_1_49_0_new/boost/thread/xtime.hpp	2015-03-03 03:54:39.741739949 +0100
@@ -20,7 +20,7 @@ namespace boost {
 
 enum xtime_clock_types
 {
-    TIME_UTC=1
+    TIME_UTC_=1
 //    TIME_TAI,
 //    TIME_MONOTONIC,
 //    TIME_PROCESS,
@@ -68,7 +68,7 @@ inline xtime get_xtime(boost::system_tim
 
 inline int xtime_get(struct xtime* xtp, int clock_type)
 {
-    if (clock_type == TIME_UTC)
+    if (clock_type == TIME_UTC_)
     {
         *xtp=get_xtime(get_system_time());
         return clock_type;
diff -rupN boost_1_49_0/libs/locale/src/icu/formatter.cpp boost_1_49_0_new/libs/locale/src/icu/formatter.cpp
--- boost_1_49_0/libs/locale/src/icu/formatter.cpp	2011-07-12 15:57:36.000000000 +0200
+++ boost_1_49_0_new/libs/locale/src/icu/formatter.cpp	2015-03-03 04:05:11.593796496 +0100
@@ -58,7 +58,7 @@ namespace locale {
             virtual string_type format(int64_t value,size_t &code_points) const
             {
                 icu::UnicodeString tmp;
-                icu_fmt_->format(value,tmp);
+                icu_fmt_->format(::int64_t(value),tmp);
                 code_points=tmp.countChar32();
                 return cvt_.std(tmp);
             }
diff -rupN boost_1_49_0/libs/thread/src/pthread/timeconv.inl boost_1_49_0_new/libs/thread/src/pthread/timeconv.inl
--- boost_1_49_0/libs/thread/src/pthread/timeconv.inl	2012-01-03 18:31:50.000000000 +0100
+++ boost_1_49_0_new/libs/thread/src/pthread/timeconv.inl	2015-03-03 04:03:30.235070745 +0100
@@ -20,8 +20,8 @@ const int NANOSECONDS_PER_MICROSECOND =
 inline void to_time(int milliseconds, boost::xtime& xt)
 {
     int res = 0;
-    res = boost::xtime_get(&xt, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC); (void)res;
+    res = boost::xtime_get(&xt, boost::TIME_UTC_);
+    BOOST_ASSERT(res == boost::TIME_UTC_); (void)res;
 
     xt.sec += (milliseconds / MILLISECONDS_PER_SECOND);
     xt.nsec += ((milliseconds % MILLISECONDS_PER_SECOND) *
@@ -56,8 +56,8 @@ inline void to_timespec_duration(const b
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC); (void)res;
+    res = boost::xtime_get(&cur, boost::TIME_UTC_);
+    BOOST_ASSERT(res == boost::TIME_UTC_); (void)res;
 
     if (boost::xtime_cmp(xt, cur) <= 0)
     {
@@ -87,8 +87,8 @@ inline void to_duration(boost::xtime xt,
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC); (void)res;
+    res = boost::xtime_get(&cur, boost::TIME_UTC_);
+    BOOST_ASSERT(res == boost::TIME_UTC_); (void)res;
 
     if (boost::xtime_cmp(xt, cur) <= 0)
         milliseconds = 0;
@@ -109,8 +109,8 @@ inline void to_microduration(boost::xtim
 {
     boost::xtime cur;
     int res = 0;
-    res = boost::xtime_get(&cur, boost::TIME_UTC);
-    BOOST_ASSERT(res == boost::TIME_UTC); (void)res;
+    res = boost::xtime_get(&cur, boost::TIME_UTC_);
+    BOOST_ASSERT(res == boost::TIME_UTC_); (void)res;
 
     if (boost::xtime_cmp(xt, cur) <= 0)
         microseconds = 0;

